acestep/api_training_routes.py

```
@@ -0,0 +1,253 @@
+
+def _wrap_response(data: Any, code: int = 200, error: Optional[str] = None) -> Dict[str, Any]:
+    import time
+    return {
+        "data": data,
+        "code": code,
+        "error": error,
+        "timestamp": int(time.time() * 1000),
+    }
+
+@router.get("/v1/training/status")
+async def api_training_status(authorization: Optional[str] = Header(None)):
```

Ruff (0.14.14)

[warning] Unused function argument: `authorization`

(ARG001)